packages/synapse-react-client/src/components/widgets/query-filter/CombinedRangeFacetFilter.tsx

```diff
@@ -104,19 +136,19 @@ export const CombinedRangeFacetFilter: React.FunctionComponent<
           }
         ></RadioGroup>
         {radioValue === RadioValuesEnum.RANGE &&
-          (col1Min === col2Max ? (
-            <label>{col2Max}</label>
+          (col1MinMin === col2MaxMax ? (
+            <label>{col2MaxMax}</label>
           ) : (
             <>
               {columnType === 'INTEGER' && (
                 <RangeSlider
-                  key="RangeSlider"
-                  domain={[col1Min, col2Max]}
+                  key={`RangeSlider-${selectedMin}-${selectedMax}`}
+                  domain={[col1MinMin, col2MaxMax]}
                   initialValues={{ min: selectedMin, max: selectedMax }}
```

Comment:
Set the key to force re-rendering when selectedMin and selectedMax changes.  At times it did not re-render, even though the initialValues were updated to the new selected values!